main.go

```diff
@@ -19,7 +19,8 @@ var (
 	dnsAddr     = flag.String("lookupd-dns-address", "", "The nsqlookupd DNS entry")
 	cfgAddr     = flag.String("config-http-address", "", "The config address")
 	httpAddrCfg = flag.Bool("config-addresses-as-http", false, "Config nsqlookupd http addresses")
-)
```

Comment:
can we make sure to run `go fmt` on the file. it looks like the formatting is off